ntfs-3g.yaml

```diff
@@ -74,55 +93,70 @@ subpackages:
   - name: ntfs-3g-libs
     pipeline:
       - runs: |
-          mkdir -p ${{targets.subpkgdir}}/lib/
-          mv ${{targets.destdir}}/lib/*.so.* ${{targets.subpkgdir}}/lib/
+          mkdir -p ${{targets.subpkgdir}}/usr/lib/
+          mv ${{targets.destdir}}/usr/lib/*.so.* ${{targets.subpkgdir}}/usr/lib/
     test:
       pipeline:
         - uses: test/ldd-check
           with:
             packages: $(basename ${{targets.contextdir}})
+    dependencies:
+      runtime:
+        - merged-sbin
+        - wolfi-baselayout
 
   - name: ntfs-3g-progs
+    dependencies:
+      provider-priority: 10
+      provides:
+        - ntfs-3g
```

Comment:
please remove provider-priority and provides.

Reply:
done